docs/user_guide.md

```diff
@@ -317,7 +335,7 @@ optional arguments:
                         directory. (By default, CodeChecker would keep reports
                         and overwrites only those files that were update by
                         the current build command).
-  -n NAME, --name NAME  Annotate the ran analysis with a custom name in the
+  -n NAME, --name NAME  Annotate the run analysis with a custom name in the
```

Comment:
I don't think this change is grammatically correct.

Reply:
I think, it is. This 3rd form makes an adjective from the verb. (I only know its Hungarian name :smile: )

Reply:
My bad. Not having actually *learnt* the language for around a decade can make some bad habits burn in. 🙂